src/components/Dialog-Window/Tabs/DialogTabs.tsx

```diff
@@ -21,6 +21,52 @@ type Translation = {
   text: string;
 };
 
+const useItemInfo = (item: CommonItemType): {
+  hasText: boolean;
+  hasLinks: boolean;
+  hasVideo: boolean;
+  hasAudio: boolean;
+  showNote: boolean;
+  showTabs: boolean;
+} => {
+  const { description, topicImage, dialog } = item;
+
+  if (!dialog) {
+    return {
+      hasText: false,
+      hasLinks: false,
+      hasVideo: false,
+      hasAudio: false,
+      showNote: false,
+      showTabs: false,
+    };
+  }
+
+  const smallScreen = useMedia('(max-width: 768px)');
+  const showNote = dialog.hasNote && smallScreen;
+
+  const hasText = !!(dialog.text || topicImage || description);
+  const hasLinks =
+    !!((dialog.links &&
+      dialog.links?.filter((link) => Boolean(link.url)).length > 0) ||
+      dialog.showAddLinks);
+  const hasVideo = !!(dialog.video?.[0]?.path);
+  const hasAudio = !!(dialog.audio?.audioFile?.[0]?.path);
+
+  // Only show tabs if there is more than one item to choose from
+  const numberOfContentItems = [hasText, hasLinks, hasVideo, hasAudio, showNote].filter(Boolean).length;
+  const showTabs = numberOfContentItems > 1;
```

Comment:
I'm not sure if this only looks smart or if it helps readability, but:

```suggestion
  const showTabs = [hasText, hasLinks, hasVideo, hasAudio, showNote].some(Boolean);
```